_data/other.yml

```
@@ -116,6 +116,15 @@ websites:
     tfa:
       - totp
 
+  - name: Clover
+    url: https://www.clover.com
+    img: clover.png
+    tfa:
+      - totp
```

```suggestion
      - totp
      - sms
```
The doc page mentions SMS 2FA as well.